nucleus/payara-modules/nucleus-microprofile/config-service/src/main/java/fish/payara/nucleus/microprofile/config/spi/ConfigProviderResolverImpl.java

```diff
@@ -368,6 +373,18 @@ public String getDeployedApplicationProperty(String applicationName, String name
         return result;
     }
 
+    public List<Properties> getDeployedApplicationAliasProperties(String applicationName) {
+        ApplicationInfo info = applicationRegistry.get(applicationName);
+        List<Properties> result = Collections.EMPTY_LIST;
```

Comment:
`Collections.emptyList()` is suggested to be preferred over using `Collections.EMPTY_LIST` directly.

Reply:
Was copied from another method. I'll make the change there too